keyboards/cozykeys/bloomer/readme.md

```diff
@@ -0,0 +1,28 @@
+# Bloomer
+
+![Bloomer](http://assets.cozykeys.xyz/images/keyboards/bloomer/bloomer-angle-2_1600x1600.jpg)
+
+- Keyboard Maintainer: [Paul Ewing](https://github.com/pcewing)
+- Hardware Supported: ItsyBitsy 32u4 5V 16MHz
+- Hardware Availability: [Bloomer Repository](https://github.com/cozykeys/bloomer)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cozykeys/bloomer/v2:default  # for Bloomer v2
+    make cozykeys/bloomer/v3:default  # for Bloomer v3
+
+Flashing example for this keyboard:
+
+    make cozykeys/bloomer/v2:default:flash  # for Bloomer v2
+    make cozykeys/bloomer/v3:default:flash  # for Bloomer v3
+
+See the
+[build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and
+the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for
+more information. Brand new to QMK? Start with our
+[Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Flashing the Firmware
+
+See the QMK docs for handwiring a keyboard; there is a section with
+instructions on how to flash the *.hex* file to the Teensy 2.0 controller.
```

Comment:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Flashing the Firmware

See the QMK docs for handwiring a keyboard; there is a section with instructions on how to flash the *.hex* file to the Teensy 2.0 controller.
```

Reply:
These render to identical HTML and I like keeping line length short as is plays more nicely with my editor (vim). Is this necessary?

Reply:
In general, we don't hard wrap lines - screens are large enough these days that it shouldn't be needed.